app/packages/core/src/Dataset.ts

```diff
@@ -132,17 +134,6 @@ export const DatasetNodeQuery = graphql`
       viewCls
       viewName
       savedViewSlug
-      appConfig {
```

Comment:
Isn't this different from appConfig in dataset?

Reply:
not sure what appConfig you are referring to? `appConfig` here is defined twice in the same query. Is that intensional?

Reply:
You are right @kaixi-wang that this is duplicate code. A query like this is actually bad, and should be broken out into smaller fragments for better consumption. Not important right now

But there is a global app config in the GraphQL schema. It is actually not used by the App (yet) because it is initialized from the event source (subscription) which is not yet GraphQL